### PR TITLE
LPS-84808 Allows rendering components without a wrapper element

### PR DESCRIPTION
**Pull request by julien**

### CODE REVIEW

**Comment by julien** [comment]
ci:test

File: modules/apps/frontend-taglib/frontend-taglib-soy/src/main/java/com/liferay/frontend/taglib/soy/internal/util/SoyJavaScriptRendererUtil.java [hunk 1]
```diff
@@ -32,17 +32,21 @@
 public class SoyJavaScriptRendererUtil {
 
 	public static String getJavaScript(
-		Map<String, Object> context, String id, Set<String> modules) {
+		Map<String, Object> context, String id, Set<String> modules,
```
Comment (jbalsas) [inline modules/apps/frontend-taglib/frontend-taglib-soy/src/main/java/com/liferay/frontend/taglib/soy/internal/util/SoyJavaScriptRendererUtil.java]: This is a breaking change. We can't do this. You should overload this method to keep the default behaviour

File: modules/apps/frontend-taglib/frontend-taglib-soy/src/main/java/com/liferay/frontend/taglib/soy/internal/util/SoyJavaScriptRendererUtil.java [hunk 2]
```diff
@@ -32,17 +32,21 @@
...
+		boolean useWrapper) {
 
 		JSONSerializer jsonSerializer = JSONFactoryUtil.createJSONSerializer();
 
 		String contextString = jsonSerializer.serializeDeep(context);
 
 		String modulesString = jsonSerializer.serialize(modules);
 
+		String wrapperString = jsonSerializer.serialize(useWrapper);
```
Comment (jbalsas) [inline modules/apps/frontend-taglib/frontend-taglib-soy/src/main/java/com/liferay/frontend/taglib/soy/internal/util/SoyJavaScriptRendererUtil.java]: Do we really need to serialize a boolean?

File: modules/apps/frontend-taglib/frontend-taglib-soy/src/main/java/com/liferay/frontend/taglib/soy/servlet/taglib/TemplateRendererTag.java [hunk 3]
```diff
@@ -233,7 +242,7 @@ protected void renderJavaScript(
 		}
 
 		String componentJavaScript = SoyJavaScriptRendererUtil.getJavaScript(
-			context, getComponentId(), requiredModules);
+			context, getComponentId(), requiredModules, isRenderWrapper());
```
Comment (jbalsas) [inline modules/apps/frontend-taglib/frontend-taglib-soy/src/main/java/com/liferay/frontend/taglib/soy/servlet/taglib/TemplateRendererTag.java]: This seems to be used in several places below here. Cache the value up here and just use it afterwards.

**Pull request closed without merging**